django_netjsonconfig/tests/test_vpn.py

```diff
@@ -223,9 +225,9 @@ def test_dh(self):
         self.assertTrue(v.dh.startswith('-----BEGIN DH PARAMETERS-----'))
         self.assertTrue(v.dh.endswith('-----END DH PARAMETERS-----\n'))
 
-    def test_context_empty(self):
+    def test_context_empty_vpn(self):
```

Comment:
I changed name of this function, because old name would lead to misunderstanding, it now clearly say that we are testing empty vpn object